README.md

```diff
@@ -223,14 +214,15 @@ You can either deploy the dbp fully or choose the components you need for your s
 By default the dbp chart will deploy fully.
 
 To disable specific components you can set the following values to false when deploying:
-
+```bash 
```

Comment:
Don't think you need 'bash' here, just the three ticks.

Reply:
Fixed.